File: chit_chat_backend/models/user_model.js


class UserModel {
    constructor(user_id, username, email, password, phone, registration_date) {
        this.user_id = user_id;
        this.username = username;
        this.email = email;
        this.password = password;
        this.phone = phone;
        this.registration_date = registration_date;
    }
}


module.exports = UserModel;

File: chit_chat/lib/models/user_model.dart
class UserModel {
  String user_id;
  String username;
  String email;
  String? phone;
  String? registration_date;
  String? img;

  UserModel(
      {required this.user_id,
      required this.username,
      required this.email,
      this.phone,
      this.registration_date,
      this.img});
}
